epoxy-sample/src/main/java/com/airbnb/epoxy/sample/models/ColorModel.java

```diff
@@ -17,14 +24,83 @@
  * anything in onBind.
  */
 @EpoxyModelClass(layout = R.layout.model_color)
-public abstract class ColorModel extends EpoxyModel<View> {
+public abstract class ColorModel extends EpoxyModelWithHolder<ColorHolder> {
   @EpoxyAttribute @ColorInt int color;
   @EpoxyAttribute int carousel;
+  @EpoxyAttribute boolean playAnimation;
   @EpoxyAttribute(DoNotHash) View.OnClickListener clickListener;
 
   @Override
-  public void bind(View view) {
-    view.setBackgroundColor(color);
-    view.setOnClickListener(clickListener);
+  public void bind(ColorHolder holder) {
+    holder.cardView.setBackgroundColor(color);
+    holder.cardView.setOnClickListener(clickListener);
+  }
+
+  @Override
+  public void bind(ColorHolder holder, EpoxyModel<?> previouslyBoundModel) {
+    // When this model changes we get a bind call with the previously bound model, so we can see
+    // what changed and update accordingly.
+    ColorModel previousModel = (ColorModel) previouslyBoundModel;
+    if (previousModel.playAnimation != playAnimation) {
+      toggleAnimation(holder.lottieView, playAnimation);
+    } else {
+      bind(holder);
+    }
+  }
+
+  private void toggleAnimation(final LottieAnimationView lottieView, boolean playAnimation) {
+    if (!playAnimation) {
+      if (lottieView.isAnimating()) {
+        // Reverse it just for fun
+        lottieView.resumeReverseAnimation();
+      } else {
+        cancelAnimation(lottieView);
+      }
+      return;
+    }
+
+    lottieView.addAnimatorListener(new AnimatorListener() {
```

Comment:
`SimpleAnimatorListener` can be used instead of `AnimatorListener` to avoid redundant override of `onAnimationRepeat`.